internal/pkg/aws/iam/iam.go

```diff
@@ -98,6 +99,21 @@ func (c *IAM) CreateECSServiceLinkedRole() error {
 	return nil
 }
 
+// ListPermBoundPolicyNames returns a list of permissions boundary policy names.
+func (c *IAM) ListPermBoundPolicyNames() ([]*string, error) {
+	policies, err := c.client.ListPolicies(&iam.ListPoliciesInput{
```

Comment:
We might need to handle pagination here: https://docs.aws.amazon.com/sdk-for-go/api/service/iam/#ListPoliciesOutput

Reply:
Yeah. I was hoping the `PolicyUsageFilter: aws.String("PermissionsBoundary")` filter would reduce the number to a reasonable amount, but I just tested it and you're right. Will add!

Reply:
yeah basically we need to do pagination for every list API we call as long as the response is paginated and we return the full list of it (instead of grabbing first xyz items)